Gemfile.lock

```diff
@@ -74,7 +74,7 @@ GEM
     addressable (2.8.0)
       public_suffix (>= 2.0.2, < 5.0)
     ast (2.4.2)
-    brakeman (5.0.4)
+    brakeman (5.1.1)
```

Comment:
Changes to dependencies in `Gemfile.lock` shouldn't be included in a PR to change a component unless those dependencies are related to the change - which in this case I think they're not. Can these changes be removed?